src/universal/components/AsyncEditorSuggestions.js

```diff
@@ -0,0 +1,49 @@
+import {css} from 'aphrodite-local-styles/no-important';
+import PropTypes from 'prop-types';
+import React from 'react';
+import ui from 'universal/styles/ui';
+import withStyles from 'universal/styles/withStyles';
+
+const dontTellDraft = (e) => {
+  e.preventDefault();
+};
+
+const AsyncEditorSuggestions = (props) => {
+  const {
+    activeIdx,
+    handleSelect,
+    styles,
+    suggestions,
+    SuggestionItem
+  } = props;
+
+  const menuStyles = css(styles.mentionMenu);
+  return (
+    <div className={menuStyles}>
+      {suggestions && suggestions.map((suggestion, idx) => {
+        return (
+          // eslint-disable-next-line
+          <div key={idx} onMouseDown={dontTellDraft} onClick={handleSelect(idx)}>
```

Comment:
the plain button still has some styles that i don't want since here, since the child is non deterministic.  I should make an `PlainListItem` component!